backend/src/main/scala/cromwell/backend/standard/StandardFinalizationActor.scala

```diff
@@ -0,0 +1,36 @@
+package cromwell.backend.standard
+
+import cromwell.backend._
+import cromwell.core.CallOutputs
+import wdl4s.TaskCall
+
+trait StandardFinalizationActorParams {
+  val workflowDescriptor: BackendWorkflowDescriptor
+  val calls: Set[TaskCall]
+  val jobExecutionMap: JobExecutionMap
+  val workflowOutputs: CallOutputs
+  val initializationDataOption: Option[BackendInitializationData]
+  val configurationDescriptor: BackendConfigurationDescriptor
+}
+
+case class DefaultStandardFinalizationActorParams
+(
+  workflowDescriptor: BackendWorkflowDescriptor,
+  calls: Set[TaskCall],
+  jobExecutionMap: JobExecutionMap,
+  workflowOutputs: CallOutputs,
+  initializationDataOption: Option[BackendInitializationData],
+  configurationDescriptor: BackendConfigurationDescriptor
+) extends StandardFinalizationActorParams
+
+trait StandardFinalizationActor extends BackendWorkflowFinalizationActor {
+  /** Standard set of parameters passed to the backend. */
```

Comment:
I'm mainly just opposed to this pattern:

    // does XYZ
    // def doXyz()

which isn't what's here, but it's in the family tree. :) Feel free to ignore